docs/vereisten/beginsel_van_proportionaliteit_en_subsidiariteit.md

```diff
@@ -50,9 +50,8 @@ Gegevensverwerking moet in verhouding staan tot het beoogde doel en persoonsgege
 
 ## Toelichting 
 
-Proportionaliteit vereist dat de omvang van gegevensverwerking voor de ontwikkeling van een algoritmes of AI en voor het genereren van de benodigde output in balans is met het beoogde doel.
-Subsidiariteit benadrukt dat persoonsgegevens alleen moeten worden verwerkt als dit de enige geschikte manier is om het doel te bereiken.
-Deze principes waarborgen dat de privacy van individuen wordt gerespecteerd en dat gegevensverwerking niet verder gaat dan noodzakelijk is voor legitieme doeleinden.
+Proportionaliteit vereist dat de impact van gegevensverwerking op de persoonlijke levenssfeer voor de toepassing van een algoritme of AI en voor het genereren van de benodigde output in balans is met het beoogde doel.
+Subsidiariteit benadrukt dat het doel op de minst vergaande manier moet worden bereikt. Dat betekent soms dat een verwerking helemaal niet hoeft plaats te vinden omdat het doel op een andere manier bereikt kan worden of soms dat er privacy by design maatregelen kunnen worden toegepast om de verwerking zelf te minimaliseren. 
```

Comment:
Subsidiariteit benadrukt dat persoonsgegevens alleen moeten worden verwerkt als dit de enige geschikte manier is om het doel te bereiken.